packages/babel-preset-mc-app/index.js

```diff
@@ -143,7 +149,7 @@ module.exports = function getBabePresetConfigForMcApp(api, opts = {}) {
         {
           // corejs messes with jest@27 in tests, due to some
           // Promise/Date related polyfills it provides.
-          corejs: !isEnvTest && 3,
+          corejs: isEnvTest || options.disableCoreJs ? false : 3,
```

Comment:
This would enable `corejs` in test env all the time regardless of the `disableCoreJs` option.

Reply:
You mean disable, right? If so yes, that's what @pa3 implemented prior and intended.